time_service.rb

```diff
@@ -0,0 +1,42 @@
+require 'rack'
+require 'hoodoo'
+
+
+#This is the implementation class. 
+#the implementation class is refered to by the interface class therefore, Implementation class must be defined first.
+#naming convention: FooImplementation.
+class TimeImplementation < Hoodoo::Services::Implementation
+    def show( context )
+      context.response.set_resource( { 'time' => Time.now.utc.iso8601 } )
+    end
```

Comment:
We are quite fussy about the formatting in our code, because it helps keep things consistent.
We don't yet have any tooling to help this, but for now can you change your editor settings to:

-  indent `2` spaces for `*.rb` files
- Remove any trailing spaces at the end of lines